src/test/fixtures/profiles/processed-profile.js

```diff
@@ -769,6 +770,7 @@ type IPCMarkersOptions = {|
   messageSeqno: number,
   side: 'parent' | 'child',
   direction: 'sending' | 'receiving',
+  phase: 'endpoint' | 'transferBegin' | 'transferEnd',
```

Comment:
I'm debating whether I should change `transferBegin` to `transferStart`, since that lines up more closely with `startTime`. That'd require a change to the (not-yet-landed) m-c patch though, and `transferBegin` has more of a ring to it to me (though that could be from my C++ background with `begin()`/`end()`).

Reply:
I went ahead and did this.